.tmux.conf.local

```diff
@@ -242,6 +243,12 @@ tmux_conf_theme_prefix_fg="none"
 tmux_conf_theme_prefix_bg="none"
 tmux_conf_theme_prefix_attr="none"
 
+# mouse indicator
+tmux_conf_theme_mouse='🖱️'            # U+1F5B1
```

Comment:
I prefer U+2191 as Emoi tend to trigger status line bugs when different components disagree on Unicode character width
```
# mouse indicator
tmux_conf_theme_mouse="↗"                   # U+2197
tmux_conf_theme_mouse_fg="none"
tmux_conf_theme_mouse_bg="none"
tmux_conf_theme_mouse_attr="none"
```